packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -888,9 +890,32 @@ function PartnerLogo( { className }: { className?: string } ) {
 	/* eslint-enable wpcalypso/jsx-classname-namespace */
 }
 
-function DomainDiscountCallout( { product }: { product: ResponseCartProduct } ) {
+function DomainDiscountCallout( {
+	product,
+	responseCart,
+	selectedSiteData,
+}: {
+	product: ResponseCartProduct;
+	responseCart: ResponseCart;
+	selectedSiteData: SiteDetails | undefined | null;
+} ) {
 	const translate = useTranslate();
 
+	const planInCart = responseCart.products.find( ( product ) => isPlan( product ) );
+	const currentSitePlan = selectedSiteData?.plan?.product_slug;
+
+	if ( planInCart ) {
+		if ( product.is_bundled && is100YearPlan( planInCart.product_slug ) ) {
```

Comment:
These two if statements could be combined.

```suggestion
	if ( planInCart && product.is_bundled && is100YearPlan( planInCart.product_slug ) ) {
```